pkg/scheduler/scheduler.go

```diff
@@ -371,15 +371,13 @@ var defaultQueueingHintFn = func(_ klog.Logger, _ *v1.Pod, _, _ interface{}) (fr
 	return framework.Queue, nil
 }
 
-func buildQueueingHintMap(es []framework.EnqueueExtensions) internalqueue.QueueingHintMap {
+func buildQueueingHintMap(logger klog.Logger, es []framework.EnqueueExtensions) internalqueue.QueueingHintMap {
 	queueingHintMap := make(internalqueue.QueueingHintMap)
 	for _, e := range es {
 		events := e.EventsToRegister()
 
-		// This will happen when plugin registers with empty events, it's usually the case a pod
-		// will become reschedulable only for self-update, e.g. schedulingGates plugin, the pod
-		// will enter into the activeQ via priorityQueue.Update().
 		if len(events) == 0 {
+			logger.V(2).Info("no event registered", "plugin", e.Name())
```

Comment:
Should we change it to error when QHint is enabled? 
I don't come up with any usecase to register no event, that is, I believe that's mistake of users most likely.

Reply:
let's keep it as it is then. It's not a big concern to me.

Reply:
I mean, can we just change it to an error log without making it fail?

Reply:
hmm.. I believe it's not a good practice to print an error but actually do nothing. I was intend to make it a warning, but right now warning is replaced with info level.

Reply:
Okay, makes sense. Let's keep it then. Can we elaborate the message more at least; 

```suggestion
			logger.V(2).Info("no event registered in EventsToRegister, which may result in Pods rejected by this plugin stuck in the unschedulable pod pool", "plugin", e.Name())
```

Reply:
Will change after https://github.com/kubernetes/kubernetes/pull/125527 merged.